### DISCUSSION TITLE
Looking forward to contribute the project

### DISCUSSION THREAD

**Discussion by JOSHUAJEBARAJ** [aquasecurity/trivy#1311] [Development]
Hello ,
I have been using the **trivy** for the last 3 months. I have been amazed by the capabilities and easiness the trivy brings to the security :). I am looking forward to contributing to the trivy project , I have gone through the open issues but couldn't find any good first issue or help needed tag, I am can help to the trivy by the below ways
- Documentation
- Writing Tests
- Implementing small features

I am looking forward to hearing from the team :)

**Comment by itaysk** [comment]
Hi @JOSHUAJEBARAJ , sorry for missing this. we would love your contribution in any way possible. 
There are some "Good First Issue"s here: https://github.com/aquasecurity/trivy/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
Also feel free to add unsolicited PRs around tests and docs improvements if you find opportunities to improve.
If you need more interactive guidance you can try contacting us in Slack (https://slack.aquasec.com) in the `#trivy` channel
